en/micro-integrator/docs/develop/injecting-parameters.md

```diff
@@ -191,6 +191,9 @@ In the following example, parameters are configured as dynamic values in the dat
 
 The <b>pinned servers</b> parameter can be dynamically injected to a scheduled task or proxy service. See the example given below.
 
+!!! Note
+    The pinned server parameter is applicable only when in non-clustering mode.
```

Comment:
By 'non-clustering mode', I presume we mean 'coordination disabled'? If so, may I suggest the following:

"The pinned server parameter is only applicable when you have a [clustered deployment](https://ei.docs.wso2.com/en/latest/micro-integrator/setup/deployment/deploying_wso2_ei/) with [coordination](https://ei.docs.wso2.com/en/latest/micro-integrator/setup/deployment/deploying_wso2_ei/#cluster-coordination) enabled."

wdyt?
@Nirothipan

Reply:
Isn't it the opposite?

"The pinned server parameter is only applicable when you have a clustered deployment with coordination disabled."

However, I assumed it could also be a single node, so "non-clustering mode" seemed to cover all the bases.

Reply:
yes.. it should be

"The pinned server parameter is only applicable when you have a clustered deployment with coordination disabled."

The trouble with "non-clustering mode" is that there is nothing called 'clustering mode' really. We don't enable/disable clustering. If we have a 'cluster of nodes', we can choose to enable coordination. That is how the clustering doc is written, which is why I suggested this change.

Reply:
This is not applicable to EI 7.1.0 which has a different implementation than EI 6. For scheduled tasks, if pinned server is defined it will run only in pinned server, so no need of this note

Reply:
Updated the text. 

So, this change should only be done in Ei 6.x family docs?

Reply:
yes